lewis_emulators/tti355/device.py

```diff
@@ -21,6 +21,8 @@ def _initialize_data(self):
         self._max_voltage = 35.0
         self._max_current = 5.0
         self.load_resistance = 10
+        self.min_voltage = None
+        self.min_current = None
```

Comment:
As I said in the other PR, can probably put the sim/max stuff in here rather than None

Reply:
Utilised sub-classing to provide extra check with the ttiex355p emulator rather than using the tti355